README.md

```diff
@@ -306,12 +306,14 @@ The configuration options for the Kafka Connect source connector for IBM MQ are
 | `mq.reconnect.delay.min.ms`                     | The minimum delay (in milliseconds) for reconnecting to the queue manager after a connection error.                                                                     | long    | 64             | 1 or greater                                                                                                                                                                        |
 | `mq.reconnect.delay.max.ms`                     | The maximum delay (in milliseconds) for reconnecting to the queue manager after a connection error.                                                                     | long    | 8192           | 1 or greater                                                                                                                                                                        |
 | `mq.receive.max.poll.time.ms`                   | Maximum time (in milliseconds) to poll messages in a single Kafka Connect task cycle. If set to 0, polling continues until batch size or a receive returns null.        | long    | 0              | 0 or greater                                                                                                                                                                        |
 | `errors.deadletterqueue.topic.name`             | The name of the Kafka topic to use as the dead letter queue (DLQ) for poison messages that fail during processing within the record builder component of the connector. | string  |                | If left blank (default), failed messages will not be written to a DLQ.                                                                                                              |
 | `errors.deadletterqueue.context.headers.enable` | Whether to add error context headers to messages written to the DLQ.                                                                                                    | boolean | false          | When enabled, additional headers describing the error will be included with each DLQ record.                                                                                        |
+| `mq.record.builder.json.schemas.enable`         | (JSON record builder only) Include schemas within Kafka messages. This is used as the `schemas.enable` config for the JsonConverter used by the JSON record builder     | boolean | false          |                                                                                                                                                                                     |
```

Comment:
Should we mention that when enabling this config its required to provide the schema content or else you will end up with `org.apache.kafka.connect.errors.DataException: Converting byte[] to Kafka Connect data failed due to serialization error: 
`